src/core/trulens/core/feedback/feedback.py

```diff
@@ -152,10 +152,36 @@ class Feedback(feedback_schema.FeedbackDefinition):
     [FeedbackDefinition.aggregator][trulens.core.schema.feedback.FeedbackDefinition.aggregator].
     """
 
+    examples: Optional[List[Tuple]] = pydantic.Field(None, exclude=True)
+    """Examples to use when evaluating the feedback function."""
+
+    min_score_val: Optional[int] = pydantic.Field(None, exclude=True)
+    """Minimum score value for the feedback function."""
+
+    max_score_val: Optional[int] = pydantic.Field(None, exclude=True)
+    """Maximum score value for the feedback function."""
+
+    temperature: Optional[float] = pydantic.Field(None, exclude=True)
+    """Temperature parameter for the feedback function."""
+
+    use_sent_tokenize: Optional[bool] = pydantic.Field(None, exclude=True)
```

Comment:
for `use_sent_tokenize` and `filter_trivial_statements`, wdyt if we can encapsulate them in a separate (pydantic) class called sth like GroundednessConfigs ?  this way we have composition and don't have to include these 2 (and potentially more) groundedness-specific attributes.